services/movie_session.py

```diff
@@ -0,0 +1,63 @@
+from db.models import MovieSession, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    movi_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=cinema_hall,
+        movie_id=movie_id
+    )
+
+    return movi_session
+
+
+def get_movies_sessions(
+        session_date: str = None
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]